apis/python/tests/test_dataframe_index_columns.py

```diff
@@ -1812,70 +1812,100 @@ def test_types_write_errors(
             ["int32"],
             None,
             [pa.array([3202, 3203])],
-            RuntimeError,  # Static type (INT64) does not match expected type (INT32)
+            (
+                tiledb.cc.TileDBError,
```

Comment:
not caused by this PR, but necessary today in my sandbox (either on `main` or this branch)